platform/admin/ui/src/main/webapp/templates/configuration/textTypeList.handlebars

```diff
@@ -12,7 +12,20 @@
  **/
  --}}
  <td>
-    <input type='text' name='value' class="{{id}} metatype" id='{{id}}' value='{{value}}'>
+     {{#if optionValues}}
+         {{#is optionValues.length 0}}
+             <input type="{{#is type 1}}text{{/is}}{{#is type 5}}text{{/is}}{{#is type 6}}text{{/is}}{{#is type 7}}text{{/is}}{{#is type 8}}text{{/is}}{{#is type 9}}text{{/is}}{{#is type 10}}text{{/is}}{{#is type 12}}password{{/is}}{{#is type 2}}number{{/is}}{{#is type 3}}number{{/is}}{{#is type 4}}number{{/is}}" class="metatype" name="value" value="{{value}}">
+         {{/is}}
+         {{#isnt optionValues.length 0}}
+             <select name="value" class="form-control input-sm">
+                 {{#each optionValues}}
+                     <option value="{{.}}" {{#is this ../value}}selected="selected" {{/is}}>{{lookup ../optionLabels @index}}</option>
+                 {{/each}}
+             </select>
+         {{/isnt}}
+     {{else}}
+         <input type="{{#is type 1}}text{{/is}}{{#is type 5}}text{{/is}}{{#is type 6}}text{{/is}}{{#is type 7}}text{{/is}}{{#is type 8}}text{{/is}}{{#is type 9}}text{{/is}}{{#is type 10}}text{{/is}}{{#is type 12}}password{{/is}}{{#is type 2}}number{{/is}}{{#is type 3}}number{{/is}}{{#is type 4}}number{{/is}}" class="metatype" name="value" value="{{value}}">
```

Comment:
✏️ A comment here would be nice to explain these magic numbers. Also, should you include an `else` condition?